app/src/ui/dialog/dialog.tsx

```diff
@@ -526,17 +526,18 @@ export class Dialog extends React.Component<IDialogProps, IDialogState> {
     if (!e) {
       if (this.dialogElement) {
         this.dialogElement.removeEventListener('cancel', this.onDialogCancel)
-        this.dialogElement.removeEventListener('keydown', this.onKeyDown)
       }
     } else {
       e.addEventListener('cancel', this.onDialogCancel)
-      e.addEventListener('keydown', this.onKeyDown)
```

Comment:
So happy to get rid of this!

Reply:
I bet, too bad cancel and others are nowhere near that easily done